external-api/src/v1/controllers/estore/eStoreApi.ts

```diff
@@ -63,46 +64,54 @@ const postToEstore = async (
 };
 
 export const siteExists = async (exporterName: EstoreSite): Promise<SiteExistsResponse> => {
-  const timeout = 1000 * 30; // 30 seconds timeout to handle long timeouts
-  const response = await postToEstore(`site/exist`, [exporterName], timeout);
+  console.info('Checking if a site exists');
```

Comment:
Please change to `'Checking if a site exists for exporter %s', $expoterName`. This will help debugging.

Reply:
updated now